components/layout/layout_task.rs

```diff
@@ -865,6 +874,114 @@ impl LayoutTask {
         rw_data.client_rect_response = iterator.client_rect;
     }
 
+    fn process_resolved_style_request<'a>(&'a self,
+                                          requested_node: TrustedNodeAddress,
+                                          property: &Atom,
+                                          //pseudo: &PseudoElementType,
+                                          layout_root: &mut FlowRef,
+                                          rw_data: &mut RWGuard<'a>) {
+        // FIXME: Isolate this transmutation into a "bridge" module.
+        // FIXME(rust#16366): The following line had to be moved because of a
+        // rustc bug. It should be in the next unsafe block.
+        let node: LayoutJS<Node> = unsafe {
+            LayoutJS::from_trusted_node_address(requested_node)
+        };
+        let node: &LayoutNode = unsafe {
+            transmute(&node)
+        };
+
+        let layout_node = ThreadSafeLayoutNode::new(node);
+        /*let layout_node = match pseudo {
+            PseudoElementType::Before => node.get_before_pseudo().unwrap(),
+            PseudoElementType::After => node.get_after_pseudo().unwrap(),
+            _ => layout_node
+        };*/
```

Comment:
Just say `FIXME: Implement pseudo-elements` here—I'm not a fan of commented-out code.